### PR TITLE
feat(locale): add states for Croatia (hr)

### PR DESCRIPTION
**Pull request by matthewmayer**
Part of #1977

There are 21 counties

https://en.wikipedia.org/wiki/Counties_of_Croatia
https://hr.wikipedia.org/wiki/Hrvatske_%C5%BEupanije

### CODE REVIEW

File: src/locales/hr/location/state.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  'Bjelovarsko-bilogorska',
+  'Brodsko-posavska',
+  'Dubrovačko-neretvanska',
+  'Grad Zagreb',
+  'Istarska',
+  'Karlovačka',
+  'Koprivničko-križevačka',
+  'Krapinsko-zagorska',
+  'Ličko-senjska',
+  'Međimurska',
```
Comment (ST-DDT) [inline src/locales/hr/location/state.ts:11]: According to the pages you have linked I could only find: [ Međimurska županija](https://hr.wikipedia.org/wiki/Me%C4%91imurska_%C5%BEupanija) which I would call a county.

grafik

**Review by ST-DDT — COMMENTED** [review]
It might be best to just call them counties here.
As for states we could use these https://en.wikipedia.org/wiki/Regions_of_Croatia or omit them entirely.

File: src/locales/hr/location/state.ts (hunk 1 above)
Comment (matthewmayer) [reply src/locales/hr/location/state.ts:11]: https://en.wikipedia.org/wiki/ISO_3166-2:HR ISO 3166-2 uses the counties as the 1st level administrative subdivisions, the regions seem to be more historical and not included in a typical address if i am not mistaken, so i feel the counties should be under `state()`

Comment (ST-DDT) [reply src/locales/hr/location/state.ts:11]: While technically it is the first level adminstrative subdivision, I would still use them as counties and simply mark state as being not applicable due to them having county as their name.

**Comment by ST-DDT** [comment]
**Team Decision**

We will jump to a definition loophole and will put it in both state (as first level administrative unit) and in county (as county names).

We will create a separate issue to clarify that states ~= `ISO_3166-2`

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Please put these in both state and county. County should import state.

**Review by ST-DDT — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**